### PR TITLE
Add panX

### PR DESCRIPTION
**Pull request by shayanjamal**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/panX/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "panx" %}
+{% set version = "0.0.0" %}
```
Comment (druvus) [inline recipes/panX/meta.yaml]: Could you ask the authors to tag a release?

Comment (bebatut) [reply recipes/panX/meta.yaml]: We already asked them. ping @wdingx

**Comment by bgruening** [comment]
Upstream does not seem to have an issue tracker activated. Thats unfortunate. @druvus what do you think about merging it in a few days if @wdingx does not respond?

**Comment by wdingx** [comment]
@druvus @bebatut @bgruening  thank you very much for your contribution. The current version is v1.5

**Comment by bgruening** [comment]
@wdingx thanks for your response. Any change to get an official release? https://github.com/neherlab/pan-genome-analysis/releases

File: recipes/panX/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% set md5 = "11247d01528e8ac1a4e85df75f38e5fd" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  #fn: {{ version }}.tar.gz
```
Comment (druvus) [inline recipes/panX/meta.yaml]: please add the file name as `panx_{{ version }}.tar.gz`

**Comment by bgruening** [comment]
Thanks everyone!

**Pull request merged**